<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialog</class>
 <widget class="QDialog" name="dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>361</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>日期时间</string>
   </property>
   <widget class="QPushButton" name="btnGetTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>读取当前日期时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>字符串显示</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>定时器</string>
    </property>
   </widget>
   <widget class="QLabel" name="labElapsTime">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>流逝时间</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>181</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>日期时间</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QTimeEdit" name="timeEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>时间</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>日期</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QDateEdit" name="dateEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLCDNumber" name="lcdHour"/>
     </item>
     <item row="0" column="2">
      <widget class="QLCDNumber" name="lcdSec"/>
     </item>
     <item row="0" column="1">
      <widget class="QLCDNumber" name="lcdMin"/>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="editTime">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>61</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDate">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>86</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDateTime">
    <property name="geometry">
     <rect>
      <x>209</x>
      <y>111</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStop">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>172</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>172</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>203</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>定时周期</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxIntv">
       <property name="suffix">
        <string>ms</string>
       </property>
       <property name="maximum">
        <number>10000</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSetIntv">
       <property name="text">
        <string>设置周期</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>311</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>日历</string>
   </property>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>296</width>
      <height>218</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>291</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>calendarWidget</tabstop>
  <tabstop>btnClose</tabstop>
  <tabstop>timeEdit</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>btnGetTime</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>editTime</tabstop>
  <tabstop>editDate</tabstop>
  <tabstop>editDateTime</tabstop>
  <tabstop>btnStop</tabstop>
  <tabstop>btnStart</tabstop>
  <tabstop>btnSetIntv</tabstop>
  <tabstop>spinBoxIntv</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
